examples/eks_managed_node_group/main.tf

```diff
@@ -421,6 +424,24 @@ module "vpc" {
   tags = local.tags
 }
 
+module "vpc_cni_irsa" {
+  source  = "terraform-aws-modules/iam/aws//modules/iam-role-for-service-accounts-eks"
+  version = "~> 4.12"
+
+  role_name_prefix      = "VPC-CNI-IRSA"
+  attach_vpc_cni_policy = true
+  vpc_cni_enable_ipv6   = true
+
+  oidc_providers = {
+    ex = {
```

Comment:
`ex` is what you called `main` in another module (`iam`), right? `main` seems more descriptive to me.

Reply:
fixed all references in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1882/commits/94c0309a7fd35174ab77462bf509e03fc60a27b3